sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateBoxDao.java

```diff
@@ -41,10 +41,11 @@
 @Repository
 public class HibernateBoxDao implements BoxStore, HibernatePaginatedDataSource<Box> {
 
-  public static final String[] SEARCH_PROPERTIES = new String[] { "name", "alias", "identificationBarcode", "locationBarcode" };
+  private static final String FIELD_ALIAS = "alias";
 
-  private final static List<String> STANDARD_ALIASES = Arrays.asList("lastModifier",
-      "creator", "size", "use");
+  protected static final String[] SEARCH_PROPERTIES = new String[] { "name", FIELD_ALIAS, "identificationBarcode", "locationBarcode" };
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "identificationBarcode" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)